Neos.Neos/Resources/Private/Templates/Module/Management/Workspaces/Show.html

```diff
@@ -109,52 +109,54 @@
 			<f:form action="index" id="postHelper" method="post"></f:form>
 
 			<script>
-				(function($) {
-					$('#check-all').change(function() {
-						var value = false;
-						if ($(this).is(':checked')) {
-							value = true;
-							$('.batch-action').removeClass('neos-hidden').removeClass('neos-disabled').removeAttr('disabled');
-						} else {
-							$('.batch-action').addClass('neos-hidden').addClass('neos-disabled').attr('disabled', 'disabled');
-						}
-						$('tbody input[type="checkbox"]').prop('checked', value);
-					});
+				window.addEventListener('DOMContentLoaded', (event) => {
+					jQuery(function() {
```

Comment:
Adjust also the related code.

```suggestion
					jQuery(function($) {
```